surveys/surveys.go

```diff
@@ -99,10 +98,11 @@ func GetAvailableSchemas() LauncherSchemas {
 			LauncherSchemaFromFilename("mbs_0867.json"),
 			LauncherSchemaFromFilename("mbs_0873.json"),
 			LauncherSchemaFromFilename("mci_transformation.json"),
```

Comment:
Technically these aren't live yet, I was thinking, should have 3 drop downs...i.e `Business`/`Social`/`Test`

Reply:
This wasn't recorded in the Trello card. When was this decided?

Reply:
we talked about it at a tech session, but you're right should have been on the ticket

Reply:
This is however on the card.